axlearn/cloud/gcp/job.py

```diff
@@ -443,6 +444,7 @@ class Config(GKEJob.Config):
         enable_tpu_ici_resiliency: Optional[bool] = None
         location_hint: Optional[str] = None
         enable_tpu_smart_repair: bool = False
+        import_pathways: Optional[list[str]] = []
```

Comment:
The dynamic imports do not seem limited to pathways:

```suggestion
        import_modules: list[str] = []
```